images/chaos-daemon/Dockerfile

```diff
@@ -1,23 +1,37 @@
-FROM curlimages/curl:7.88.1 as binaryimage
-
-USER root
+FROM debian:bullseye-slim
 
 ARG HTTPS_PROXY
 ARG HTTP_PROXY
 
 ENV http_proxy $HTTP_PROXY
 ENV https_proxy $HTTPS_PROXY
 
+ENV BYTEMAN_HOME /usr/local/byteman
+ENV PATH $PATH:/usr/local/byteman/bin
+ENV JAVA_HOME /usr/lib/jvm/java-11-openjdk-amd64
+ENV PATH $PATH:/usr/local/bin
+
+RUN mkdir -p /usr/share/man/man1 /usr/share/man/man2
+
+RUN apt-get update && \
+    echo "deb http://security.debian.org/debian-security bullseye-security main contrib non-free" >> /etc/apt/sources.list && \
+    echo "deb http://deb.debian.org/debian bullseye-proposed-updates main contrib non-free" >> /etc/apt/sources.list && \
+    apt --allow-releaseinfo-change update && apt upgrade  -y && \
+    apt-get install -y tzdata iptables ebtables net-tools ipset stress-ng iproute2 fuse util-linux procps curl openjdk-11-jre && \
```

Comment:
I guess the only difference between the output of this and using the curl image for the first step is that this image will have `curl` available in chaos-daemon instances?

Reply:
Yes. But also the dashboard image uses the same one so we don't need to build 2 different ones internally, just the debian is enough